testing/features/support/components/base.rb

```diff
@@ -6,6 +6,10 @@ class Base < SitePrism::Page
     [validate_initial_state!, "Initial #{self.class} component didn't load correctly!"]
   end
 
+  def switch_language(_language)
+    "TBD"
```

Comment:
This is the only non-functional piece of this PR.

Reply:
Still figuring out an straight forward solution on this. Will get back to you on it soon.